build.xml

```diff
@@ -43,6 +63,189 @@
     </java>
   </target>
   
+  <target name="preclean" description="Clean out the results of any previous runs">
+    <description>
+      TARGET preClean
+      This target is designed to clean out the results of any previous processing
+      runs so that they don't get in the way or get re-transformed.
+    </description>
+    
+    <dirname property="ploddDir" file="${inputPlodd}"/>
+    <echo message="Cleaning out prior build detritus from ${ploddDir}."/>
```

Comment:
Should be "Cleaning out detritus from XXX prior to build itself" or "Prior to build itself, clean out detritus from XXX".